cfgov/core/lib/PDFreactor.py

```diff
@@ -0,0 +1,2268 @@
+"""This is the PDFreactor 6.3 Python API 
```

Comment:
Does `cfgov/core/lib/` need to be added to the repo or can the libs be downloaded at build time?